Classes/Core/Functional/Framework/Frontend/Response.php

```diff
@@ -19,7 +19,6 @@
  */
 class Response
 {
-    const STATUS_Success = 'success';
```

Comment:
Not sure about changes in the `Response` layer... probably currently not used, but might be useful in future™

Reply:
We could/should use status codes instead, once we need it